Assets/Temp/EnemyAITest.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyAITest : MonoBehaviour
+{
+    [SerializeField]
+    private float _Health;
+    [SerializeField]
+    private float _Speed = 5;
```

Comment:
The naming convention is:
_variable for private
Variable for properties
variable for everything else